specs/Media.md

```diff
@@ -0,0 +1,193 @@
+# Background
+WebView2 is a control that developers can implement for part of, or for the entirety of their app. In the either case, developers may need the ability to control the media within the webview's bounds. This API will allow developers to control the sound of each webview2 instance by enabling the mute/unmute of the media within the webview. There will also be the ability to check if the sound is muted or audible.
+
+# Description
+Enable end developer to `Mute`, `Unmute` a webview content. And ability to check for the `IsMuted` and `IsCurrentlyAudible`.
+
+# Examples
+
+The following code snippet demonstrates how the Media related API can be used:
+
+## Win32 C++
+
+```cpp
+AudioComponent::AudioComponent(AppWindow* appWindow)
+    : m_appWindow(appWindow), m_webView(appWindow->GetWebView())
+{
+    //! [IsCurrentlyAudibleChanged]
+    // Register a handler for the IsCurrentlyAudibleChanged event.
+    // This handler just announces the audible state on the window's title bar.
+    CHECK_FAILURE(m_webView->add_IsCurrentlyAudibleChanged(
+        Callback<ICoreWebView2StagingIsCurrentlyAudibleChangedEventHandler>(
+            [this](ICoreWebView2Staging2* sender, IUnknown* args) -> HRESULT {
+                BOOL isAudible;
```

Comment:
These two event handlers are copies of one another. Probably want to move out to one method that you call from both.